### PR TITLE
review: docs

### PR DESCRIPTION
**Pull request by byeongsu-hong**

### CODE REVIEW

File: example/TestRecipient.bin [hunk 1]
```diff
@@ -0,0 +1 @@
...565b95989497509550505050565b60008060006040848603121561061657600080fd5b83359250602084013567ffffffffffffffff81111561063457600080fd5b61064086828701610553565b9497909650939450505050565b600181811c9082168061066157607f821691505b60208210810361068157634e487b7160e01b600052602260045260246000fd5b50919050565b81835281816020850137506000828201602090810191909152601f909101601f19169091010190565b6020815260006106c4602083018486610687565b949350505050565b634e487b7160e01b600052604160045260246000fd5b601f82111561072e576000816000526020600020601f850160051c8101602086101561070b5750805b601f850160051c820191505b8181101561072a57828155600101610717565b5050505b505050565b67ffffffffffffffff83111561074b5761074b6106cc565b61075f83610759835461064d565b836106e2565b6000601f841160018114610793576000851561077b5750838201355b600019600387901b1c1916600186901b178355610310565b600083815260209020601f19861690835b828110156107c457868501358255602094850194600190920191016107a4565b50868210156107e15760001960f88860031b161c19848701351681555b505060018560011b0183555050505050565b83815260406020820152600061080d604083018486610687565b9594505050505056fea2646970667358221220e74b9ce655ed9c3a34678c7f1f825221c112607de021b8a463cd377e3305ee6564736f6c63430008170033
```
Comment (nambrot) [inline example/TestRecipient.bin:1]: Can we import this from the SDK?

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
+# Deploying Hyperlane with Osmosis Testnet
```
Comment (nambrot) [inline example/README.md]: Let's rename this to DEPLOYMENT.md or something.

File: scripts/GUIDE_DEPLOY.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
+# Deploy Guide with LocalOsmosis
```
Comment (nambrot) [inline scripts/GUIDE_DEPLOY.md]: What is the difference between the example readme and this? Just that it is local osmosis? There is probably a lot of overlap?

File: example/README.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+## 1. Create `config.yaml` with your network config
+
+> Don't forget to setup deploy settings below
+
+Below is an example of `config.yaml` file for localwasmd.
+
+You can check full list of example in [config.example.yaml](../config.example.yaml) file.
+
+```yaml
```
Comment (nambrot) [inline example/README.md]: Does the ISM not have to be configured here? Don't see the command down below either?

Comment (nambrot) [reply example/README.md]: Or if it happens, it's very implicit

Comment (byeongsu-hong) [reply example/README.md]: I agree about it - it's very implicit. The multisig ism with one validator (deployer), 1 / 1 threshold if the deployer configure ism with just array of domains.

We can remove the `default` action and add some validation logic for undefined ism.

**Review by hashableric — APPROVED** [review]

File: example/TestRecipient.bin (hunk 1 above)
Comment (byeongsu-hong) [reply example/TestRecipient.bin:1]: [58e04ca](https://github.com/many-things/cw-hyperlane/pull/96/commits/58e04caf88a23369acdf2bcd6bf7d440eb80ffeb) this commit from pr #96 will do it

File: scripts/GUIDE_DEPLOY.md (hunk 3 above)
Comment (byeongsu-hong) [reply scripts/GUIDE_DEPLOY.md]: Oh this file shouldn't be there. will remove it

File: example/README.md (hunk 4 above)
Comment (byeongsu-hong) [reply example/README.md]: @nambrot I'll fix this at #100

**Pull request merged**